Source/WebKit/Shared/GoToBackForwardItemParameters.h

```diff
@@ -43,7 +43,7 @@ struct GoToBackForwardItemParameters {
     std::optional<WebsitePoliciesData> websitePolicies;
     bool lastNavigationWasAppInitiated;
     std::optional<NetworkResourceLoadIdentifier> existingNetworkResourceLoadIdentifierToResume;
-    std::optional<String> topPrivatelyControlledDomain;
+    String publicSuffix;
```

Comment:
Thanks for cleaning this up! I was wondering if we could make a change like this after my last PR.